docs/UserGuide.md

```diff
@@ -420,102 +422,76 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
 
 ### Find leave records by time period: `find-leave-range`
 
-* **What It Does:**
-  * Finds all leave records in HRMate in a given time period. There are 4 possible scenarios:
-  * 1) The time period has a start date and an end date (inclusive) - all leaves with at least one day that falls within
-      this period will be displayed
-    2) The time period only has a start date (inclusive) - all leaves that either occur on the start date or will occur
-      after the start date will be displayed
-    3) The time period only has an end date (inclusive) - all leaves that either occur on the end date or will finish
-      before the end date will be displayed
-    4) The time period does not have a start date or end date - all leaves will be returned
-* **Format:**
-  * `find-leave-range [start/START_DATE] [end/END_DATE]`
-* **Examples:**
+If you wish to view the leave records in HRMate that fall within a particular time period, 
+you can use the `find-leave-range` command to do so.
 
-| S/N | Command information                                                                                                                                                                     |
-|-----|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `find-leave-range start/2023-10-27 end/2023-11-03` <br><br> **Output:** Returns all leaves that have at least one day in the period of 2023-10-27 and 2023-11-03 inclusive |
-| 2   | **Command:** `find-leave-range start/2023-10-27` <br><br> **Output:** Returns all leaves that have at least one day either on or after 2023-10-27                                       |
-| 3   | **Command:** `find-leave-range end/2023-11-03` <br><br> **Output:** Returns all leaves that have at least one day on or before 2023-11-03                                               |
-| 4   | **Command:** `find-leave-range` <br><br> **Output:** Returns all leaves                                                                                                                 |
+1. Type in the following [command](#glossary) in the [command box](#glossary):
+    `find-leave-range [start/START_DATE] [end/END_DATE]`, and replace `START_DATE` with the start date of the time period,
+    and `END_DATE` with the end date of the time period, both in the format `yyyy-MM-dd`.
```

Comment:
Might be good to include a section to explain what yyyy-MM-dd means / a link to some section that explains it